index.html

```diff
@@ -18,24 +19,68 @@
 </head>
 <body>
 
-<div class="container-fluid">
-    <h1>EUMIS Sumission management</h1>
+  <nav class="navbar navbar-inverse navbar-fixed-top">
+    <div class="container">
+        <div class="col-sm-6">
+            <img src="images/logo.jpg" id="se-logo">
+        </div>
+        <div class="col-sm-6">
+            <a class="navbar-brand" href="#">EUMIS Submissions</a>
+        </div>
+      </div>
+      <!--<div id="navbar" class="collapse navbar-collapse">
```

Comment:
Don't commit commented out content. If it's needed it's needed.  If it's not. It's not.